src/include/dump_dir.h

```diff
@@ -346,7 +346,7 @@ int dd_set_no_owner(struct dump_dir *dd);
  * If meta-data misses owner, returns fs owner.
  * Can be used with DD_OPEN_FD_ONLY.
  */
-uid_t dd_get_owner(struct dump_dir *dd);
+int dd_get_owner(struct dump_dir *dd, uid_t *owner);
```

Comment:
Wouldn't it be better to return the `uid_t` result and have some kind of `errorno` parameter? 
That way we could use the function directly and test the value of `errorno` for errors. And if we don't care about errors just pass it `NULL`.